logs/system.log

```diff
@@ -0,0 +1,42 @@
+22:49:26.577 [main] INFO  com.todolist.app.service.ToDoStoreSingleton - New task was added!
```

Comment:
I don't understand why we need it. Usually repo should contain only code in addition to some configuration files.